tokens/src/core/global/typography.json

```diff
@@ -2,90 +2,284 @@
-        "small": {
-          "base": { "value": "87.5%", "type": "percentage", "source": "$small-font-size", "description": "Small font size." },
-          "x": { "value": "75%", "type": "percentage", "source": "$x-small-font-size", "description": "X-small font size." }
-        },
-        "h1": { "value": "2.5rem", "type": "dimension", "source": "$h1-font-size", "description": "Font size of heading level 1." },
-        "h2": { "value": "2rem", "type": "dimension", "source": "$h2-font-size", "description": "Font size of heading level 2." },
-        "h3": { "value": "1.375rem", "type": "dimension", "source": "$h3-font-size", "description": "Font size of heading level 3." },
-        "h4": { "value": "1.125rem", "type": "dimension", "source": "$h4-font-size", "description": "Font size of heading level 4." },
-        "h5": { "value": ".875rem", "type": "dimension", "source": "$h5-font-size", "description": "Font size of heading level 5." },
-        "h6": { "value": ".75rem", "type": "dimension", "source": "$h6-font-size", "description": "Font size of heading level 6." },
-        "mobile": {
-          "h1": { "value": "2.25rem", "type": "dimension", "source": "$h1-mobile-font-size", "description": "Mobile font size of heading level 1." },
-          "h2": { "value": "{typography.font.size.h2}", "type": "dimension", "source": "$h2-mobile-font-size", "description": "Mobile font size of heading level 2." },
-          "h3": { "value": "{typography.font.size.h3}", "type": "dimension", "source": "$h3-mobile-font-size", "description": "Mobile font size of heading level 3." },
-          "h4": { "value": "{typography.font.size.h4}", "type": "dimension", "source": "$h4-mobile-font-size", "description": "Mobile font size of heading level 4." },
-          "h5": { "value": "{typography.font.size.h5}", "type": "dimension", "source": "$h5-mobile-font-size", "description": "Mobile font size of heading level 5." },
-          "h6": { "value": "{typography.font.size.h6}", "type": "dimension", "source": "$h6-mobile-font-size", "description": "Mobile font size of heading level 6." }
+        "$type": "dimension",
+        "base": {
+          "source": "$font-size-base",
+          "$value": "1.125rem",
+          "$description": "Base font size."
         },
-        "lead": {
-          "value": "calc({typography.font.size.base} * 1.25)",
-          "type": "dimension",
+        "lg": {
           "source": "$lead-font-size",
-          "description": "Lead text font size."
+          "$value": "{typography.font.size.base} * 1.25",
+          "$description": "Lead text font size."
+        },
+        "sm": {
+          "source": "$small-font-size",
+          "$value": ".875rem",
+          "$description": "Small font size."
+        },
+        "xs": {
+          "source": "$x-small-font-size",
+          "$value": ".75rem",
+          "$description": "X-small font size."
```

Comment:
Comparing the computed typography values in the docs for this PR's local checkout vs. published docs site, it looks like we might actually want/need to keep the `%` scale these tokens originally had.

For example, `x-small` used to be 75% but we migrated it to `.75rem`, which seems to result in slightly different font sizes. Note the `.small` and `.x-small` examples specifically (left: local; right: prod):

<img width="1899" alt="image" src="https://github.com/user-attachments/assets/0aa2dd69-9662-4b22-a0f1-9222ac081f81">

[aside] Related, you might notice the line heights are also slightly off for some of them as well. This seems to be due to the switch to `DataTable`, where some of the cell content is inheriting `DataTable`'s line-height styles, i.e. `.pgn__data-table td` (left: local; right: prod):

<img width="1911" alt="image" src="https://github.com/user-attachments/assets/f01d3623-b099-4537-9d01-a447b030dcd0">

I suppose we could consider overriding `DataTable`'s default CSS styles for typography example tables by `unset`'ing the line-height related definition(s), or maybe cross-check whether the CSS utility classes are defining explicit `line-height` style properties vs. inheriting from its parent.

Reply:
Yes, we will definitely need to do a full check of all tokens after the latest changes, as we can easily miss some small detail that can have a significant impact on both the components and the documentation site. Thanks! 👍

Reply:
[inform] Implemented in follow-up commit: [6676868 (#3186)](https://github.com/openedx/paragon/pull/3186/commits/6676868e6c46d70d91f0bdc42b7c2848e3b0aa7c)